web/src/main/webapp/index.html

```diff
@@ -1,47 +1,54 @@
+<!--
+Copyright 2016 The Johns Hopkins University Applied Physics Laboratory LLC
+All rights reserved.
+ 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
 <html>
    <head>
+      <link rel="stylesheet" href="resources/css/custom.css">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
       <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-      <title> Dorset Frontend </title>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"/></script>
+      <script src="resources/js/custom.js"></script>
+      <title> Dorset Web Demo </title>
    </head>
-   <body>
-      <h2>
-         Welcome to the Dorset Front End Demo!
-      </h2>
-      <form>
-         Enter Question:<br>
-         <input type="text" name="question" id="questionInputId">          
-         <button type="button" onclick="submitQuestion()">Submit</button><br>
-         Answer: <br>
-         <input type="text" name="answer" id="answerOutputId">
-      </form>
+   <body id="background">
+         <div class="container">
+         	<br><br><br><br><br><br>
+            <div class="row">
+               <div class="col-md-6 col-md-offset-3">
+                  <div class="panel panel-default panel-transparent">
+                     <div class="panel-body">
+                        <div class="row">
+                           <div class="col-md-10 col-md-offset-1">
+                              <h2>Dorset Web Demo...</h2>
+                              <div class="form-group">
+                                 <input id="questionInputId" type="text" class="form-control" aria-describedby="basic-addon1">
+                              </div>
+                              <div class="form-group">
+                                 <textarea id="answerOutputId" class="form-control" rows="3" aria-describedby="basic-addon1" readonly></textarea>
+                              </div>
+                              <div id="controls" class="form-group centered">
+                                 <button id="start_button"  onclick="submitQuestion()" type="button" class="btn btn-info btn-lg pull-right">Ask Question</button>
```

Comment:
Not a huge issue, but you are switching between camel case and snake case (questionInputId vs start_button). Since bootstrap uses dashes, I'd stick with that (question-input-id).